initScripts/x86_64/macOS_10.12/README.md

```diff
@@ -0,0 +1,38 @@
+# macOS 10.12 Sierra
+
+Shippable builds on `macOS 10.12` could be run by adding macOS nodes as [custom nodes in Shippable](http://docs.shippable.com/platform/tutorial/runtime/custom-nodes/).
```

Comment:
macOS 10.12 nodes can be initialised with the Shippable agent to run runSh builds. Some of the pre-requisites for installing this are listed below. After installing the pre-requisites, head over to docs page to find out how to add the node using Shippable UI. Note: we do not support auto initialisation for macOS at this time.